Extension/src/LanguageServer/extension.ts

```diff
@@ -108,6 +108,13 @@ function onActivationEvent(): void {
 function realActivation(): void {
     if (new CppSettings().intelliSenseEngine === "Disabled") {
         throw new Error("Do not activate the extension when IntelliSense is disabled.");
+    } else {
+        let clangCommandAdapterActive: boolean = vscode.extensions.all.some((extension: vscode.Extension<any>, index: number, array: vscode.Extension<any>[]): boolean => {
+            return extension.isActive && extension.id === "mitaki28.vscode-clang";
+        });
+        if (clangCommandAdapterActive) {
+            vscode.window.showWarningMessage("Disable the C/C++ Clang Command Adapter extension or set C_Cpp.intelliSenseEngine to Disabled.");
```

Comment:
I don't think we should call out any one extension.  Let's gather telemetry on how big a problem this is first.